specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -100,63 +103,96 @@ namespace EventGrid {
     value: ReceiveDetails[];
   }
 
-  @doc("Failed LockToken information.")
+  @doc("Failed lock token information.")
   model FailedLockToken {
-    @doc("LockToken value")
+    @doc("The lock token of an entry in the request.")
     lockToken: string;
 
-    @doc("Error code related to the token. Example of such error codes are BadToken: which indicates the Token is not formatted correctly, TokenLost: which indicates that token is not found, and InternalServerError: For any internal server errors.")
+    @doc("Error code related to the lock token. Example of such error codes are TokenLost: which indicates that token is not found, and InternalServerError: For any internal server errors.")
     errorCode: string;
 
-    @doc("Description of the token error.")
+    @doc("Description of the lock token error.")
     errorDescription: string;
   }
 
+  @doc("Supported delays for release operation.")
```

Comment:
A numerical enum is rare - are these the only values that are allowed? E.g. if I sent the number 11 on the wire, would the service reject the call?

Reply:
In general, our guidelines recommend being explicit with the property name, like `releaseDelayInSeconds`.